server/src/test/java/org/candlepin/model/OwnerContentCuratorTest.java

```diff
@@ -513,9 +513,9 @@ public void testUpdateOwnerContentReferences() {
         Content unmodified = this.createContent("c1", "c1", owner2);
         Content updated = this.createContent("c1", "c1");
 
-        assertTrue(original.getUuid() != updated.getUuid());
-        assertTrue(original.getUuid() != unmodified.getUuid());
-        assertTrue(updated.getUuid() != unmodified.getUuid());
+        assertNotSame(original.getUuid(), updated.getUuid());
```

Comment:
These were actually incorrect entirely before, as the strings representing the UUIDs could never possibly be the same instance across those objects -- even before Content was made to be immutable at the Hibernate level.

While the updated assertion is more apt for how the test was written, it's still logically incorrect and should be using proper string comparing rather than instance checking. However, these tests (and this class as a whole) has nothing to do with the PR.